style.css

```diff
@@ -0,0 +1,6 @@
+.basis {
+  font-family: "Roboto", serif;
+  font-weight: 900;
+  height: 2766px;
+  width: 1440px;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/IFW.2/pullRequest?prid=6245874)